pkg/nuclide/server/scripts/utils.py

```diff
@@ -46,7 +46,7 @@ def http_get(host, port, method, url, key_file=None, cert_file=None, timeout=1):
             return ret
         else:
             return None
-    except:
+    except Exception:
```

Comment:
via @ssorallen: This seems reasonable but changes the behavior of this except; it will no longer catch KeyboardInterrupts nor SystemExits.